pkg/virtctl/root.go

```diff
@@ -131,6 +134,17 @@ func NewVirtctlCommandFn() (*cobra.Command, clientcmd.ClientConfig) {
 	return rootCmd, clientConfig
 }
 
+func addGLogVerbosityFlag(fs *pflag.FlagSet) {
+	// The glog verbosity flag is added to the default flag set
+	// by init() in vendor/github.com/golang/glog/glog.go.
+	// We re-add it here to make it available in virtctl commands.
+	if f := flag.CommandLine.Lookup("v"); f != nil {
+		fs.AddFlag(pflag.PFlagFromGoFlag(f))
```

Comment:
Silly question: is it valid a log verbosity of 0? i.e., `-v=0`.

Reply:
It is, but this is all encapsulated within glog. This change only makes the flag accessible through virtctl again.

Reply:
Ah okay, got it. Thanks for the clarification.